### ISSUE TITLE
Move current language to dropdown option

### ISSUE THREAD

**Issue by kyleplo** [LLK/scratch-vm#1851]
### Expected Behavior

"language" is a separate block

### Actual Behavior

It should be a dropdown option. Otherwise, Scratchers might discriminate others by not allowing a project to be run by people who speak certain languages, like this:

![image](https://user-images.githubusercontent.com/31634240/50053772-a1f40200-0108-11e9-832d-526508d9c43b.png)

But if it were in a menu, this would be impossible.

### Steps to Reproduce

Try it.

### Operating System and Browser

All.

**Comment by towerofnix** [comment]
Agreed - I'm just worried that if there are existing projects that use this block, or tutorials and the like, they would be broken / require updating, and we might be too close to release to have the time to make any required changes.
